api/src/main/java/org/ehcache/config/Eviction.java

```diff
@@ -22,7 +22,7 @@
 import java.util.concurrent.TimeUnit;
 
 /**
- * Utility class for getting predefined {@link EvictionVeto} and {@link EvictionPrioritizer} instances.
+ * Utility class for getting predefined {@link EvictionVeto} instance.
```

Comment:
![thank you](http://www.englishstuding.com/wp-content/uploads/2014/07/2346-1-english-grammar-premium.jpg)